ocaml/idl/datamodel.ml

```diff
@@ -4211,7 +4211,18 @@ let host_set_power_on_mode = call
           ]
   ~allowed_roles:_R_POOL_OP
   ()
-  
+
+let host_set_ssl_back_compat = call
+	~name:"set_ssl_back_compat"
+	~lifecycle:[Prototyped, rel_dundee, ""]
+	~doc:"Enable/disable SSLv3 for interoperability with older versions of XenServer"
+	~params:[
+		Ref _host, "host", "The host";
+		Bool, "enabled", "True to allow SSLv3 and ciphersuites as used in old XenServer versions";
```

Comment:
True, it would look like a field-setter... but unlike the other host_foo calls above and below.

For almost all the other host_foo calls that take a host parameter, that parameter is called "host" not "self" in datamodel.

For all but one of the other host_set_foo calls, the value parameter has a descriptive name rather than just "value".

Is consistency with field-setters more important?